### DISCUSSION TITLE
question? properties of fields in modules

### DISCUSSION THREAD

**Discussion by krakersy12** [YetiForceCompany/YetiForceCRM#16512] [General]
Hi, I have a question in which file edit the properties of fields created in modules? I'm pretty new here so sorry if I'm asking any obvious question

**Comment by apcloic** [comment]
@krakersy12 Hi, go to System settings -> Standard modules -> Edit fields
https://gitstable.yetiforce.com/index.php?module=LayoutEditor&parent=Settings&view=Index

**Reply by krakersy12** [reply]
Thanks for the answer, I need to change the length of the field, unfortunately it cannot be changed from the level of the field editor.

**Comment by rskrzypczak** [comment]
Example: https://github.com/YetiForceCompany/YetiForceCRM/issues/13102
